docs/superpowers/plans/2026-07-15-delegate-claim-anchored-issuance.md

```
@@ -0,0 +1,364 @@
+      expect(outcome.policy.kind).toBe('claim_grant_required');
+    });
+```
+
+- [ ] **Step 2: Run both fresh-path tests to verify they fail as expected**
+
+Run: `pnpm --filter @rundown-org/core exec jest lifecycle-command-service.test.ts -t "anchors fresh issuance on the claim"`
+
+Expected: FAIL. Before the fix the seam anchors the active default (`otherRunId`); the authorization gate then rejects the claim's missing `delegate-from-run` grant for that run, so `outcome.kind` is `'refused'` — the `expect(outcome.kind).toBe('delegated')` assertion fails.
+
+> The fall-through test (`falls through … when the claim's controlled run is terminal`) already PASSES before the fix — before the branch exists the seam always anchors the active default, which is exactly what it asserts. That is intended: it is a mutation-killing / invariant guard, not a red→green driver. Confirm it stays green after Step 3-4.
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)